js/dynamic.js

```diff
@@ -18,16 +18,52 @@ function displayNav() {
     humburgerNav.style.color = "#6070ff";
     humburgerNav.innerHTML = '<i class="fa fa-bars" aria-hidden="true"></i>';
   } else {
-    menu.style.display = 'block';
+    menu.style.display = "block";
     humburgerNav.style.color = "white";
     humburgerNav.innerHTML = '<i class="fas fa-times"></i>';
   }
 
-  menu.classList.toggle('menu');
+  menu.classList.toggle("menu");
 }
 
 humburgerNav.addEventListener("click", displayNav);
 
+//data storage in the browser
+
+const btn = document.getElementById("submitBtn");
+const body = document.getElementsByTagName('body')[0];
+
+console.log(localStorage.user);
+btn.addEventListener("click", processData);
+function processData() {
+  const names = document.getElementById("name").value;
+  const email = document.getElementById("email").value;
+  const mess = document.getElementById("message").value;
+  const User = {
+    name: names,
+    mail: email,
+    message: mess,
+  };
+  let storeUser = JSON.stringify(User);
+  if(localStorage.user!== undefined){
+    if(localStorage.user !== storeUser){
+      localStorage.setItem('user',storeUser);
+    }
+  } else{
+    localStorage.setItem('user',storeUser);
+  }
+}
+
+body.addEventListener("load", pageload);
+function pageload(){
+  if(localStorage.user !== undefined){
+    const textJ=localStorage.user;
+    const objJ = JSON.parse(textJ);
+    document.getElementById("name").value = objJ.name;
+    document.getElementById("email").value = objJ.mail;
+    document.getElementById("message").value = objJ.message;
+  }
+}
```

Comment:
## Great job so far 🙌

- [ ] One thing I noticed the data is stored or updated to localStorage when you change the text content of the inputs, not when you click on submit button. Please kindly fix this issue 😃. 

Please see this video gif for more information.
![Animation](https://user-images.githubusercontent.com/69011963/152640722-810814c7-4c76-407f-8080-4360e144248e.gif)